src/components/combo_box/combo_box.test.tsx

```diff
@@ -216,426 +222,468 @@ describe('EuiComboBox', () => {
-      expect(container.innerHTML).toContain(
-        'euiComboBox__inputWrap--fullWidth'
-      );
-    });
-
-    test('autoFocus', () => {
-      const { getByTestSubject } = render(
-        <EuiComboBox
-          options={options}
-          selectedOptions={[options[2], options[3]]}
-          autoFocus
-        />
-      );
-
-      expect(document.activeElement).toBe(
-        getByTestSubject('comboBoxSearchInput')
-      );
-    });
-
-    test('aria-label / aria-labelledby renders on the input, not on the wrapper', () => {
-      const { getByTestSubject } = render(
-        <EuiComboBox
-          options={options}
-          // Production usages shouldn't have both attributes,
-          // we're just combining them for testing expedience
-          aria-label="Test label"
-          aria-labelledby="test-heading-id"
-        />
-      );
-      const input = getByTestSubject('comboBoxSearchInput');
-
-      expect(input).toHaveAttribute('aria-label', 'Test label');
-      expect(input).toHaveAttribute('aria-labelledby', 'test-heading-id');
-    });
-
-    test('inputRef', () => {
-      const inputRefCallback = jest.fn();
-
-      const { getByRole } = render(
-        <EuiComboBox options={options} inputRef={inputRefCallback} />
-      );
-      expect(inputRefCallback).toHaveBeenCalledTimes(1);
-
-      expect(getByRole('combobox')).toBe(inputRefCallback.mock.calls[0][0]);
-    });
-
-    test('onSearchChange', () => {
-      const onSearchChange = jest.fn();
-      const { getByTestSubject, queryAllByRole } = render(
-        <EuiComboBox options={options} onSearchChange={onSearchChange} />
-      );
-      const input = getByTestSubject('comboBoxSearchInput');
-
-      fireEvent.change(input, { target: { value: 'no results' } });
-      expect(onSearchChange).toHaveBeenCalledWith('no results', false);
-      expect(queryAllByRole('option')).toHaveLength(0);
-
-      fireEvent.change(input, { target: { value: 'titan' } });
-      expect(onSearchChange).toHaveBeenCalledWith('titan', true);
-      expect(queryAllByRole('option')).toHaveLength(2);
-    });
-  });
-
-  it('does not show multiple checkmarks with duplicate labels', async () => {
-    const options = [
-      { label: 'Titan', key: 'titan1' },
-      { label: 'Titan', key: 'titan2' },
-      { label: 'Tethys' },
-    ];
-    const { baseElement } = render(
-      <EuiComboBox
-        singleSelection={{ asPlainText: true }}
-        options={options}
-        selectedOptions={[options[1]]}
-      />
-    );
-    await showEuiComboBoxOptions();
-
-    const dropdownOptions = baseElement.querySelectorAll(
-      '.euiFilterSelectItem'
-    );
-    expect(
-      dropdownOptions[0]!.querySelector('[data-euiicon-type="check"]')
-    ).toBeFalsy();
-    expect(
-      dropdownOptions[1]!.querySelector('[data-euiicon-type="check"]')
-    ).toBeTruthy();
-  });
+      it('renders a tooltip with applied props on focus', async () => {
```

Comment:
```suggestion
      it('renders a tooltip with applied props on keyboard navigation', async () => {
```
very optional suggestion - not sure if this is helpful at all, apologies if I'm being pedantic!